### PR TITLE
adding bert pretraining docs

### PR DESCRIPTION
**Pull request by yzhang123**
Signed-off-by: Yang Zhang <yangzhang@nvidia.com>

### CODE REVIEW

File: docs/source/nlp/bert_pretraining.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+| model.mask_prob                           | float           | Probability of masking a token in the input text during data processing                                      |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+| model.short_seq_prob                      | float           | Probability of having a sequence shorter than the maximum sequence length                                    |
++-------------------------------------------+-----------------+--------------------------------------------------------------------------------------------------------------+
+
+.. note::
+
+    For offline data preprocessing **model.tokenizer** is null.
+    For online data preprocessing **model.tokenizer** needs to be specified.
+
```
Comment (ekmb) [inline docs/source/nlp/bert_pretraining.rst:83]: maybe specify what tokenizer is going to be used and what tokenizer/vocab users would need when using this model on downstream tasks

How about adding a line here on how to use this pre-trained bert model + tokenizer for downstream tasks and maybe cross-reference in the https://github.com/NVIDIA/NeMo/blob/r1.0.0rc1/docs/source/nlp/nlp_model.rst

**Review by ekmb — CHANGES_REQUESTED** [review]
Please add a quick start section on how to load a pre-trained model once they're available

File: docs/source/nlp/bert_pretraining.rst (hunk 1 above)
Comment (yzhang123) [reply docs/source/nlp/bert_pretraining.rst:82]: done

**Comment by yzhang123** [comment]
@ekmb  also updated PretrainedModelInfo for BERT and QAModel, also added quick start guide with available models for both to docs

**Review by ekmb — APPROVED** [review]

**Pull request merged**